lms/templates/dashboard/_dashboard_credit_info.html

```diff
@@ -35,7 +32,7 @@
                 "at the {link_to_provider_site} website. Select <b>Request Credit</b> to get started.").format(
                     link_to_provider_site=provider_link,
                 )
-                credit_msg_class = "credit-request-pending-msg"
+                credit_msg_class = "credit-request-not-started-msg"
```

Comment:
was it intentional to update this style (pending -> not-started) but not the one two lines below?